internal/stage_wildcard.go

```diff
@@ -21,6 +21,11 @@ func testWildcard(stageHarness *test_case_harness.TestCaseHarness) error {
 			Input:            "car",
 			ExpectedExitCode: 1,
 		},
+		{
+			Pattern:          "c.+t",
+			Input:            "coat",
```

Comment:
I think we should include another case, testing the success scenario. 
```
Pattern: "g.+gol"
Input: "gooooooogol"
```

Reply:
Yep - we should definitely test a success + failure combo when testing something new. Don't want the test list to get too long though (might start hitting timeouts) - @andy1li maybe we could add the case @ryan-gang suggested (feel free to simplify it if you'd like) and remove the 2nd test case? The 2nd one (`c.t` matches `cot`) doesn't seem to test much more than the first one (`c.t` matches `cat`).

Reply:
@ryan-gang How does the updated pair look?